### PR TITLE
docs: update installation

### PR DESCRIPTION
**Pull request by Kriechi**

### CODE REVIEW

**Comment by mhils** [comment]
Looks great. Minor nitpick: My understanding is that 
```
You can check your system information by running: ``mitmproxy --sysinfo``
```
will never work if mitmproxy hasn't been installed properly and is not required at this point if installation was successful. Long story short, this part seems to be superfluous here. :)

**Comment by Kriechi** [comment]
> will never work if mitmproxy hasn't been installed properly

Thats the idea behind this sentence - to test if mitmproxy works properly. I guess I can reword it to better reflect the actual meaning. Any suggestions?

**Comment by Kriechi** [comment]
I made the requested changes.
